sites/next.skeleton.dev/src/examples/components/progress/Example.tsx

```diff
@@ -0,0 +1,5 @@
+import { Progress } from '@skeletonlabs/skeleton-react';
+
+export default () => {
```

Comment:
The React examples will need to be updated to follow this page-like structure we use in all existing component examples:

https://github.com/skeletonlabs/skeleton/blob/next/sites/next.skeleton.dev/src/examples/components/avatars/Example.tsx#L4

You can then import as follows:

```ts
import { Page as Example } from '@examples/components/avatars/Example.tsx';
import ExampleRaw from '@examples/components/avatars/Example.tsx?raw';
```